### PR TITLE
[CIRCLECI] RC RPM publish / RC binary link to PR comment

### PR DESCRIPTION
**Pull request by whoisxx**
## Proposed changes

- RC RPM publish: It's hard to install RC version by yum cause RPM publish job run after release major version. By adding job publish RPM when it comes to RC, it could be much easier to install/upgrade to RC version.
- RC binary link: circleci will make comment under release PR with each binary link every RC version. So that anyone interested could install by check release PR. It could be like image below
image

- and small changes: add linter test under test-pass job, remove ** to * on slack notification

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: .circleci/scripts/build_announce.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+set -euox
+
+VERSION=$(go run build/rpm/main.go version)~${CIRCLE_TAG##*-}
+
+SHORT_SHA1=${CIRCLE_SHA1:0:7}
+CIRCLE_PR_NUMBER=$(hub pr list -s open -L 10 -f "%I")
+
+PACKAGES="kcn kpn ken kscn kspn ksen kbn kgen homi"
+BAOBAB_PACKAGES="kcn kpn ken"
+#PACKAGE_PREFIX="http://packages.klaytn.net/klaytn/${VERSION}"
```
Comment (KimKyungup) [inline .circleci/scripts/build_announce.sh:11]: Is it ok not to use this commented code?

File: .circleci/scripts/build_announce.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    BAOBAB_DARWIN_PACKAGE_LINKS+="<a href="${PACKAGE_PREFIX}/$i-${VERSION}-0-darwin-10.10-amd64.tar.gz">$i</a>"
+    BAOBAB_DARWIN_PACKAGE_LINKS+=" "
+done
+
+COMMENT_ROWS="<ul><li>Linux: ${LINUX_PACKAGE_LINKS}</li><li>Darwin: ${DARWIN_PACKAGE_LINKS}</li><li>Baobab-linux: ${BAOBAB_LINUX_PACKAGE_LINKS}</li><li>Baobab-darwin: ${BAOBAB_DARWIN_PACKAGE_LINKS}</li></ul>"
+
+COMMENT_HEAD="Builds ready [${SHORT_SHA1}]"
+COMMENT_BODY="<details><summary>${COMMENT_HEAD}</summary>${COMMENT_ROWS}</details>"
+
+#POST_COMMENT_URI="https://api.github.com/repos/klaytn/klaytn/issues/${CIRCLE_PR_NUMBER}/comments"
```
Comment (KimKyungup) [inline .circleci/scripts/build_announce.sh:40]: Is it ok not to use this commented code?

File: .circleci/scripts/build_announce.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+do
+    LINUX_PACKAGE_LINKS+="<a href="${PACKAGE_PREFIX}/$i-${VERSION}-0-linux-amd64.tar.gz">$i</a>"
+    LINUX_PACKAGE_LINKS+=" "
+    DARWIN_PACKAGE_LINKS+="<a href="${PACKAGE_PREFIX}/$i-${VERSION}-0-darwin-10.10-amd64.tar.gz">$i</a>"
+    DARWIN_PACKAGE_LINKS+=" "
+done
+
+for i in ${BAOBAB_PACKAGES}
+do
+    BAOBAB_LINUX_PACKAGE_LINKS+="<a href="${PACKAGE_PREFIX}/$i-${VERSION}-0-linux-amd64.tar.gz">$i</a>"
```
Comment (aidan-kwon) [inline .circleci/scripts/build_announce.sh:29]: Does this tar file contains `k*nd` script and `k*nd.conf` file? 
If it does, you may need to change `PACKAGE_PREFIX` to another prefix for baobab files.

File: .circleci/scripts/build_announce.sh (hunk 1 above)
Comment (whoisxx) [reply .circleci/scripts/build_announce.sh]: you are right. I reflect your comment! thank you for letting me know

File: .circleci/scripts/build_announce.sh (hunk 2 above)
Comment (whoisxx) [reply .circleci/scripts/build_announce.sh]: I added commit to uncomment this line, thank you for letting me know

File: .circleci/scripts/build_announce.sh (hunk 3 above)
Comment (whoisxx) [reply .circleci/scripts/build_announce.sh]: Thanks. I just added commit to add baobab prefix on their links. It has `k*nd, k*nd.conf` as you said and link should be like this. 
http://package.klaytn.net/klaytn/v1.5.0~rc.2/kcn-baobab-v1.5.0~rc.2-0-linux-amd64.tar.gz

**Comment by aidan-kwon** [comment]
@Reviewers, take a look please

**Pull request merged**